cpu/saml21/include/periph_cpu.h

```diff
@@ -73,6 +73,20 @@ typedef enum {
  */
 #define DAC_NUMOF           (2)
 
+/**
+ * @name    Real time counter configuration
+ * @{
+ */
+#define RTT_DEV             RTC->MODE0
+#define RTT_IRQ             RTC_IRQn
+#define RTT_ISR             isr_rtc
+
+#define RTT_MAX_VALUE       (0xffffffff)
```

Comment:
That could be moved to `cpu/sam0_common/include/periph_cpu_common.h`

Reply:
Does it make splitting of all the definitions that much? I would rather look at everything in one header.

Reply:
Unlike `RTT_FREQUENCY`, this is not configurable but fixed for all sam0 CPUs.

Reply:
Yes, but that is not the argument I'm using, I don't want to be looking for `RTT` related  values over multiple headers. I don't think we gain anything by splitting it out.